python_modules/dagster/dagster/_core/definitions/freshness_policy.py

```diff
@@ -34,12 +34,29 @@ class FreshnessPolicy(
     cron_schedule is set), the data used to produce the current version of this asset must be no
     older than `maximum_lag_minutes` before the most recent available data.
```

Comment:
It would be helpful to be specific about what we mean by "most recent available upstream data"